Data/Boundaries/Smoothing/TolagaBay_v22p5.txt

```diff
@@ -0,0 +1,308 @@
+178.32352263 -38.31328067

```

Comment:
Unfortunately the suggested change I made earlier needs this filename to be changed to "Tolaga_Bay_v22p5.txt" now.